Source/WebKit/WebProcess/FullScreen/WebFullScreenManager.cpp

```diff
@@ -439,6 +439,7 @@ void WebFullScreenManager::setMainVideoElement(RefPtr<WebCore::HTMLVideoElement>
 
     static NeverDestroyed eventsToObserve = std::array {
         WebCore::eventNames().seekingEvent,
+        WebCore::eventNames().seekedEvent,
```

Comment:
do we also maybe wanna listen for any of these other events too?
- `"emptied"`
- `"ended"`
- `"stalled"`
- `"waiting"`

Reply:
We definitely shouldn't listen for _some_ of those — you can get stalled after seeking ends, for instance.

(Maybe we could do something upon "ended", but that would be for a different bug altogether).